packages/wells/src/index.ts

```diff
@@ -1,19 +1,18 @@
 // Copyright 2020 Cognite AS
-
 export * from '@cognite/sdk';
 export { default as CogniteWellsClient } from './client/CogniteWellsClient';
-//export { WellsAPI } from './client/api/wellsApi';
-//export { WellboresAPI } from './client/api/wellboresApi';
+export { createWellsClient } from './client/api/utils';
 
 // dataclasses and types
+export * from './client/model/Cluster';
 export * from './client/model/GeoJson';
 export * from './client/model/Well';
 export * from './client/model/Wellbore';
 export * from './client/model/WellHead';
 export * from './client/model/WellFilter';
 export * from './client/model/WellType';
 export * from './client/model/Survey';
-export * from './client/model/MeasurementType';
-export * from './client/model/Measurement';
+export { MeasurementType } from './client/model/MeasurementType';
+export { Measurement, Measurements } from './client/model/Measurement';
 export * from './constants';
```

Comment:
Javascript does not like when we export using an `*`, so I wrote them out explicitly. Seems to work in typescript though